html-syntax/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title></title>
+  <img src="https://assets.pokemon.com/assets/cms2/img/pokedex/full/009.png" alt="Blastoise" height="500" width="500">
```

Comment:
This should be in the `body` element

Reply:
Thanks Cody, I put it in the body element and committed the html-syntax exercise again. Is that the right thing to do after editing the code?